R/extensions.R

```diff
@@ -42,3 +42,18 @@ fluentPage <- function(..., suppressBootstrap = TRUE) {
     ...
   )
 }
+
+#' parseTheme
+#'
+#' Reads a theme JSON generated by Theme Designer: \url{https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/theming-designer/}
+#' and parses it to an object digestable by \code{ThemeProvider}
+#'
+#' @param json Either a JSON string or a path to JSON file containing the theme created in Theme Designer
+#' @seealso [ThemeProvider()] for usage of this function
+#' @export
+parseTheme <- function(json) {
```

Comment:
I'm fine with a single argument, given that `fromJSON()` handles it.

Still IMO: Explicit is better than implicit, and guessing whether a string is a path, URL or data is magic to be avoided. It is extremely unlikely to happen for us, but edge cases which arise in such interfaces can be really painful to deal with when they do concern you (for example, `false` is both a valid JSON string and a file path).

Reply:
Okay, I created an interface using `path` and `json` arguments ✔️